src/main/windows/main.js

```diff
@@ -45,18 +45,26 @@ function init (state, options) {
     y: initialBounds.y
   })
 
-  win.once('ready-to-show', function () {
+  win.loadURL(config.WINDOW_MAIN)
```

Comment:
Calling `win.loadURL()` does a `fs.readFile` internally to read the file from disk. That's async, so there's no way that it could fire `'ready-to-show'` on the same tick of the event loop. Since our event handler is registered on the same tick, and the event will be fired on at least the next tick, this should be safe.

Also, there are a bunch of other event handlers below this one that are registered after calling `win.loadURL`, so this better be the way things work!